app/assets/javascripts/discourse/app/components/user-menu/items-list.hbs

```diff
@@ -0,0 +1,39 @@
+{{#if this.loading}}
+  <div class="spinner-container">
+    <div class="spinner"></div>
+  </div>
+{{else if this.items.length}}
+  <ul>
+    {{#each this.items as |item|}}
+      <item.userMenuComponent @item={{item}}/>
+    {{/each}}
+  </ul>
+  <div class="panel-body-bottom">
+    {{#if this.showAll}}
+      <a class="btn btn-default btn-icon no-text show-all" href={{this.showAllHref}} title={{this.showAllTitle}}>
+        {{d-icon "chevron-down" aria-label=this.showAllTitle}}
+      </a>
+    {{/if}}
+    {{#if this.showDismiss}}
+      <button
```

Comment:
we don't want to use d-button ?

Reply:
I'm not sure. I've been trying to avoid using classic components in the new menu because of their bad performance, and there are a number of missing features* that I need in the classic `d-button` component to be able to use them here, so I don't know if I should invest in adding features to something that we'll stop using soon or if I should create a new Glimmer-based `d-button`?

* I'm saying 'features' here which may give the impression that I need to make big changes, but the truth is all I need to pass some custom classes and attributes.